pallets/asset-manager/src/lib.rs

```diff
@@ -654,6 +741,15 @@ pub mod pallet {
         pub fn check_outgoing_assets_filter(asset_location: &Option<MultiLocation>) -> bool {
             FilteredOutgoingAssetLocations::<T>::contains_key(asset_location)
         }
+
+        /// Sorted the assets pair
+        pub fn sort_asset_id(asset_0: T::AssetId, asset_1: T::AssetId) -> (T::AssetId, T::AssetId) {
+            if asset_0 < asset_1 {
+                (asset_0, asset_1)
+            } else {
+                (asset_1, asset_0)
+            }
+        }
```

Comment:
why does this matter ?

Reply:
when generate lp_asset_id, the given two asset id is already sorted, so here when we register two asset id, we should keep same order so that when query by two already sorted assetid, it will get result.